.github/workflows/testnet-release.yml

```diff
@@ -0,0 +1,32 @@
+name: "Testnet Community Release"
+
+on:
+  push: 
+    branches: 
+      - main
+env:
+  CHAIN: "picasso"
+  RELEASE_VERSION: "latest"
+
+jobs:
+  build-and-publish:
+    runs-on: 
+        - self-hosted
+        - linux
+        - x64
+        - sre
+    steps:
+    - uses: actions/checkout@v2
+      with: 
+        fetch-depth: 0
+        ref: picasso
+
+    - name: Build and Push Artifacts to gcloud
+      run: |
+        /home/runner/.cargo/bin/cargo build --release --bins
```

Comment:
Since it is triggered for each commit, can we use the `develop` feature in this case?

Reply:
This is a good idea!
Added.